common/tools/eslint-plugin-azure-sdk/src/rules/ts-package-json-sdktype.ts

```diff
@@ -26,7 +26,7 @@ export default createRule({
       ...VerifierMessages,
       SdkTypeNotString: "sdk-type is not set to a string",
       SdkTypeNotValid:
-        "unrecognized sdk-type value: {{actual}}. Expected one of 'client', 'mgmt', or 'utility.'",
+        "unrecognized sdk-type value: {{actual}}. Expected one of 'client', 'mgmt', 'perf-test', or 'utility.'",
```

Comment:
Do the warnings appear in VS Code? Our "lint" script only lints `src` for perf tests.

Reply:
Yes, they appear in the VS Code